### DISCUSSION TITLE
Help: Trying to round a join between two tubes

### DISCUSSION THREAD

**Discussion by DSchroer** [Irev-Dev/Round-Anything#37] [Q&A] [answered]
Hey, I am trying to model a lug that joins two pipes at an angle. I would like to add a fillet at the joining point that nicely runs around the join. Is there an efficient way to do it using Round-Anything? Any advice would be appreciated. 

![rounding](https://user-images.githubusercontent.com/3913937/148310939-31a5f528-6492-4844-98b5-d315bcf248f4.png)

**Comment by Irev-Dev** [comment, marked as answer]
Yeah that's hard to do in OpenSCAD. The minkowski round would work but it's so slow I'd only use it if desperate.

@TLC123 Did add some utils that might help, but I've not actually used the so don't have first hand experience. The docs for it are here https://github.com/Irev-Dev/Round-Anything/blob/master/unionRoundMask-Doc.md

Let me know how you go.

**Comment by DSchroer** [comment]
Thanks for getting back to me. I used `unionRoundMask` and it was exactly what I needed. 

![Screenshot from 2022-01-08 08-11-00](https://user-images.githubusercontent.com/3913937/148645492-bf0d51de-a385-44b2-9aeb-22905a320a78.png)

As you can see the join has a nice filet now and looks rather professional. Its a bit slower but still quite reasonable on my machine.

**Reply by Irev-Dev** [reply]
Awesome!

**Reply by DSchroer** [reply]
I put together a small tutorial modeling the part using this information. Added a shout out for your great library as well. Feel free to check it out: https://schroer.ca/2022/08/01/head-tube-lug/

**Reply by Chaphasilor** [reply]
Thanks for the write-up. Was easier to use than I thought, but it takes ages to calculate the preview on my laptop (actual rendering is reasonably fast) :)
